packages/react-devtools-extensions/chrome/manifest.json

```diff
@@ -4,48 +4,49 @@
   "description": "Adds React debugging tools to the Chrome Developer Tools.",
```

Comment:
is this commit relevant to the script in some way?

Reply:
Yes. I updated the `manifest.json` files in their own commit to match the format generated by `JSON.stringify(json, null, 2)` so the version update script would _only_ affect version numbers (and no other formatting chagnes).